cli/CHANGELOG.md

```diff
@@ -1,4 +1,12 @@
 <!-- See the ../guides/writing-the-cypress-changelog.md for details on writing the changelog. -->
+## 12.14.0
+
+_Released 06/06/2023 (PENDING)_
+
+**Features:**
+
+- Cypress now has a targeted Content-Security-Policy and Content-Security-Policy-Report-Only header directive allow list via the [`experimentalCspAllowList`](https://docs.cypress.io/guides/references/configuration#Experimental-Csp-Allow-List) configuration option. Addresses [#1030](https://github.com/cypress-io/cypress/issues/1030).
```

Comment:
I think we might need the 'Addressed in' for features to make the changelog job happy (sorry if I said otherwise 🥴 )
```suggestion
- Cypress now has a targeted Content-Security-Policy and Content-Security-Policy-Report-Only header directive allow list via the [`experimentalCspAllowList`](https://docs.cypress.io/guides/references/configuration#Experimental-Csp-Allow-List) configuration option. Addresses [#1030](https://github.com/cypress-io/cypress/issues/1030). Addressed in [#26483](https://github.com/cypress-io/cypress/pull/26483)
```